tasks.py

```diff
@@ -783,6 +792,7 @@ def assets(dev=False, watch=False):
     # on prod
     webpack(clean=False, watch=watch, dev=dev)
 
+
```

Comment:
Good to know. Found this page which may be helpful for Pycharm users looking to satisfy the built in static analysis feature (which uses the same error codes): http://iambigblind.blogspot.com/2013/02/configuring-pep8py-support-in-pycharm-27.html